var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -157,6 +161,10 @@ def configure_args(self):
                     ]
                 )
         else:
+            if "@:9.8" in spec:
+                oapp("--with-fftw {0}".format(spec["fftw-api"].prefix))
+                oapp("--with-hdf5 {0}".format(spec["hdf5"].prefix))
```

Comment:
You also need the following to get past `configure`:

```suggestion
                oapp("--with-fftw={0}".format(spec["fftw-api"].prefix))
                oapp("--with-hdf5={0}".format(spec["hdf5"].prefix))
```

Reply:
I *can* build successfully with the suggested change above.